src/java/org/apache/cassandra/utils/logging/LogbackLoggingSupport.java

```diff
@@ -105,10 +102,11 @@ public void setLoggingLevel(String classQualifier, String rawLevel) throws Excep
         // if both classQualifier and rawLevel are empty, reload from configuration
         if (StringUtils.isBlank(classQualifier) && StringUtils.isBlank(rawLevel))
         {
-            JMXConfiguratorMBean jmxConfiguratorMBean = JMX.newMBeanProxy(ManagementFactory.getPlatformMBeanServer(),
-                                                                          new ObjectName("ch.qos.logback.classic:Name=default,Type=ch.qos.logback.classic.jmx.JMXConfigurator"),
-                                                                          JMXConfiguratorMBean.class);
-            jmxConfiguratorMBean.reloadDefaultConfiguration();
+            LoggerContext lc = (LoggerContext) LoggerFactory.getILoggerFactory();
```

Comment:
that's on basis of:
https://github.com/qos-ch/logback/commit/fa3de693048d25698af7264fd294a1c9ba6940d1

we'll see how the CI is gonna look like
I'll do some manual test too.

Reply:
added a unit test that shows this works

Reply:
did the previous code register an MBean? does it do that now as well?

Reply:
Old logback had an mbean.
The new one doesn't have it.
We are no longer using it.
I don't think that mbeans registered by 3rd party libraries are part of the Cassandra interface